docker-compose.raw

```diff
@@ -0,0 +1,78 @@
+# DO NOT EDIT
+# The .env file has everything you need to edit.
+# Run options:
+# 1. Use prebuilt images (preferred method):
+#   run cmd: docker-compose up -d
+# 2. Build images on your own machine:
+#   build cmd: COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose build
+#   run cmd: docker-compose up -d
+
+version: "3.8"
+services:
+  __proxy_name__:
+    image: reallibrephotos/librephotos-proxy:${tag}
+    restart: always
+    volumes:
+      - ${myPhotos}:/data
+      - ${proMedia}:/protected_media
+      - ./proxy/nginx.conf:/etc/nginx/nginx.conf:ro
```

Comment:
This is a breaking change for existing installations. I know that a lot of people only downloaded the docker-compose.yml and the .env and didn't clone the whole repository. Is it possible to have a default where there is already a configured nginx.conf in the container when no nginx.conf is found?

Reply:
Yes there is! If you do absolutely nothing, the default container has the default container names baked in, so that'll still work.

If you want to change the names of the containers, then the `nginx.conf` file needs to be updated, and I can't think of a nice way to do that without mounting the file into the proxy container.
Keep in mind though that the original `docker-compose.yml` file still exists, and that file does _not_ have that `nginx.conf` file as a volume mount.

Reply:
Ah, right, I didn't see the ".raw" part. Looks good to me :+1: